src/modules/job-manager/job.c

```diff
@@ -148,6 +148,11 @@ void job_aux_delete (struct job *job, const void *val)
     aux_delete (&job->aux, val);
 }
 
+void job_aux_destroy (struct job *job)
+{
+    aux_destroy (&job->aux);
```

Comment:
Does there need to be a `job->aux = NULL` here, or is that implied since an empty aux_item list is `NULL`?

Reply:
Right, that's implied.